raco/backends/myria/myria.py

```diff
@@ -191,6 +191,15 @@ def compileme(self):
         }, **self.options)
         if self.format == 'OPP':
             encoding['opType'] = "SeaFlowScan"
+        elif self.format == 'TIPSY':
```

Comment:
I think you might as well just overwrite `encoding` with another assignment rather than delete attrs etc.